system/lib/wasmfs/wasmfs.h

```diff
@@ -18,6 +18,8 @@
 #include <vector>
 #include <wasi/api.h>
 
+#define WASMFS_PERM_WRITE 0222
```

Comment:
It seems odd to have this here... why is it here?

(That is - why is it here in this file, and not in the single file that currently uses it, as being here suggests it is part of the external interface. And, why is this needed at all - is there no existing define for "write permissions"?)

Reply:
Added it because the existing `stat.h` macros seemed difficult to read at a glance. Also we don't really make use of the user, group and other distinctions. It might make sense to just have a simpler macro for our use case?